CDCgov/phdi
# Adminstered medications

# PULL REQUEST

## Summary
Adds the Administered Medications Table as part of the Treatment Plan section under Clinical Info. It largely follows a similar pattern to procedures. It also assumes that the table has medication name and medication start date in the table, which most do. 

## Related Issue
Fixes #1107 

## Screenshot
<img width="642" alt="image" src="https://github.com/CDCgov/phdi/assets/107214304/f1ff8bb0-5492-48ed-a408-ab5cfea0334a">

## How to test

`cd containers/ecr-viewer`
`npm run local-dev`
go to `http://localhost:3000/view-data?id=df33f075-83a1-4d28-ba63-05bcc3339c21`
Observe the administered medications table in the clinical info section
